packages/gui/src/components/plotNFT/select/PlotNFTSelectPool.tsx

```diff
@@ -107,6 +111,19 @@ const PlotNFTSelectPool = forwardRef((props: Props, ref) => {
   }));
 
   async function handleSubmit(data: FormData) {
+    if (nfts?.length > 10) {
+       createNFT = await openDialog(
+        <ConfirmDialog
+          title="Too Many NFT's"
+          confirmColor="danger"
+        >
+          <Trans>
+            You already have more then 10 PlotNFT's, you really shouldn't make more.
+          </Trans>
+        </ConfirmDialog>
+      );
+    }
+    if (createNFT || createNFT == undefined) {
```

Comment:
With the prior suggestion of initializing `createNFT` to `true`, this can be simplified to:

```suggestion
    if (createNFT) {
```

Reply:
Added